File: CS/WindowsFormsApplication24/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTabbedMdi;
using DevExpress.XtraEditors;

namespace WindowsFormsApplication24
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			for ( int i = 0; i < 4; i++ )
			{
				XtraForm xtraForm = new XtraForm();
				xtraForm.Text = "Doc " + (i + 1);
				xtraForm.MdiParent = this;
				xtraForm.Show();
			}
		}
	}
}

File: VB/WindowsFormsApplication24/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTabbedMdi
Imports DevExpress.XtraEditors

Namespace WindowsFormsApplication24
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			For i As Integer = 0 To 3
				Dim xtraForm As New XtraForm()
				xtraForm.Text = "Doc " & (i + 1)
				xtraForm.MdiParent = Me
				xtraForm.Show()
			Next i
		End Sub
	End Class
End Namespace